openadapt/record.py

```diff
@@ -370,7 +371,7 @@ def read_window_events(
         recording_timestamp: The timestamp of the recording.
     """
 
-    utils.configure_logging(logger, LOG_LEVEL)
+    utils.configure_logging(logger, LOG_LEVEL, SCRUB_LOGGING)
```

Comment:
Rather than passing this in explicitly everywhere we call `utils.configure_logging`, how about we modify `utils.py` so that it is included as a default, e.g.:

```
def configure_logging(..., scrub_logging=SCRUB_LOGGING)
...
```